modules/nw-metallb-configure-community-bgp-advertisement.adoc

```diff
@@ -0,0 +1,111 @@
+:_content-type: PROCEDURE
+[id="nw-metallb-configure-BGP-advertisement-community-alias_{context}"]
+= Configuring MetalLB with a BGP advertisement and community alias
+
+Configure MetalLB as follows so that the `IPAddressPool` is advertised with the BGP protocol and the community alias set to the numeric value of the well-known `NO_ADVERTISE` community.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+
+* Log in as a user with `cluster-admin` privileges.
+
```

Comment:
Every section heading must have an anchor ID per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-titles-and-section-headings